api/bazel/repository_locations.bzl

```diff
@@ -16,12 +16,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "protoc-gen-validate (PGV)",
         project_desc = "protoc plugin to generate polyglot message validators",
         project_url = "https://github.com/bufbuild/protoc-gen-validate",
-        use_category = ["api"],
-        sha256 = "f1ec013cfdfffa7a17d75b55d41265dad47d24e0e9d86c02311562e15be52da9",
-        version = "1.0.1",
+        use_category = ["api", "dataplane_core"],
```

Comment:
@htuch i added this here hoping it would resolve the missing `protoc_gen_lyft_star` dep validation (missing from `dataplane_core`)

it doesnt seem to have worked, despite pgv having lyft_start as an `implied_untracked_dep` so not sure how to resolve

Reply:
Why is PGV being used in the dataplane now?

Reply:
Can you share the underlying `bazel query` that shows the dependency graph leading to these deps being reachable from the dataplane targets?

Reply:
stepping through the code, running queries and grasping straws - i think the underlying change/issue is here https://github.com/bufbuild/protoc-gen-validate/pull/900

didnt quite figure out why that broke us but closer